File: abc/abc144/b/main.py
n = int(input())

ans = "No"
for i in range(1, 10):
    if n == i:
        ans = "Yes"
        break
    if i != 1 and n % i == 0 and n / i < 10:
        ans = "Yes"
        break
print(ans)

File: abc/abc144/b/main.cc
#include <math.h>
#include <algorithm>
#include <iostream>
#include <string>
#include <vector>
#define rep(i, s, n) for (int i = s; i < (n); ++i)
using namespace std;

int main() {
  int n;
  cin >> n;

  string ans = "No";

  rep(i, 1, 10) {
    if (n == i) {
      ans = "Yes";
      break;
    }
    if (i != 1 && n % i == 0 && n / i < 10) {
      ans = "Yes";
      break;
    }
  }
  cout << ans << endl;
}